doc/040_backup.rst

```diff
@@ -204,6 +204,16 @@ Combined with ``--verbose``, you can see a list of changes:
     modified  /archive.tar.gz, saved in 0.140s (25.542 MiB added)
     Would be added to the repository: 25.551 MiB
 
+Estimator Scanner
+*****************
+
+When you start a backup, Restic will launch a separate thread to count
```

Comment:
I'd like to avoid mentioning "thread" here as its just an implementation detail, just saying that e.g. "[...], restic will concurrently to the backup count the number of files and their total size, [...]" should be enough.